src/test/java/Homework17.java

```diff
@@ -0,0 +1,56 @@
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.util.Scanner;
+
+public class Homework17 extends BaseTest{
+  //  @Test
+    //public static void CreatePlaylist() {
+
+
+
+
+       // WebElement createNewPlaylist = driver.findElement(By.xpath("//i[@title='Create a new playlist']"));
+      //  createNewPlaylist.click();
+     //   WebElement newPlaylist = driver.findElement(By.xpath("//li[@data-testid='playlist-context-menu-create-simple']"));
+      //  newPlaylist.click();
+
+      //  WebElement playlistInput = driver.findElement(By.xpath("//input[@name='name']"));
+      //  playlistInput.sendKeys("My playlist");
+
+        //I tried to find code for pressing Enter button. But I did it wrong and I don't know how to fix it
+
+       // Scanner keybord = new Scanner(System.in);
+       // keybord.nextLine();
+
+
+      //  WebElement student = driver.findElement(By.xpath("//a[text()='My playlist']"));
+      //  Assert.assertTrue(student.isDisplayed());
+   // }
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.

Reply:
I didn't know how to write code for pressing 'Enter' tab on the keyboard. I google it but still was not able to find a good solution.